scripts/sanitycheck

```diff
@@ -2528,6 +2528,10 @@ def parse_arguments():
         "-S", "--enable-slow", action="store_true",
         help="Execute time-consuming test cases that have been marked "
         "as 'slow' in testcase.yaml. Normally these are only built.")
+    parser.add_argument(
+        "--disable-unrecognized-section-test", action="store_false",
+        default=True,
```

Comment:
Shouldn't this default to `False` to keep backwards compatibility with the current behavior?

Reply:
Good catch. Something strange happened, fixing ...